README.md

```diff
@@ -66,24 +66,25 @@ client.close();
 
 ### Avaialable public methods
 ```
-const value = client.boolVariation('test', target, false);
-const value = client.stringVariation('test', target, 'BLACK');
-const value = client.numberVariation('test', target, 1);
-const value = client.jsonVariation('test', target, {});
+function boolVariation(identifier: string, target: Target, defaultValue = true): Promise<boolean>;
```

Comment:
At some point we should make these links to documentation on the specific functions and how to use them. Another time though

Reply:
make sense